src/components/Select/Select.docs.mdx

```diff
@@ -0,0 +1,32 @@
+import { Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import Select from './Select';
+
+<Meta title="Select" component={Select} />
+
+# Select
+
+Presents users with a choice of options from a dropdown.
+
+## Usage
+
+```js
+import { Select } from '@quartz/interface/src/components';
+```
```

Comment:
Don't worry about this bit, it's not really useful since we now include instructions for importing in the readme
```suggestion
```